.github/workflows/bump-version-pr.yaml

```diff
@@ -0,0 +1,72 @@
+name: bump-version-pr
+
+on:
+  workflow_dispatch:
+
+jobs:
+  bump-version-pr:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+        with:
+          ref: main
+          fetch-depth: 0
+
+      - name: Generate token
+        id: generate-token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.APP_ID }}
+          private-key: ${{ secrets.PRIVATE_KEY }}
+
+      - name: Set git config
+        uses: autowarefoundation/autoware-github-actions/set-git-config@v1
+        with:
+          token: ${{ steps.generate-token.outputs.token }}
+
+      - name: Setup Python 3.x
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.x
+
+      - name: Install dependencies
+        run: pip3 install -U catkin_tools
+        shell: bash
+
+      - name: Bump version from humble branch
+        id: bump-version-from-humble-branch
+        run: |
+          git fetch origin --tags
+          git checkout origin/humble
+          git checkout -b tmp/bot/bump_version_base
+          git fetch origin main
+          git merge origin/main
+          catkin_generate_changelog -y
+          git add *
+          git commit -m "update CHANGELOG.rst"
+          catkin_prepare_release -y --bump minor --no-push
```

Comment:
You need a mechanism where “major,” “minor,” or “patch” can be passed as an argument.

Reply:
I think the `choice` input is useful in this case.
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs

Reply:
[Done!](https://github.com/autowarefoundation/autoware.universe/pull/9990/commits/407ad8fe3d09fd9f4ec7bbbb5c10eb64caa9a2ac)
![Screenshot from 2025-01-21 14-08-10](https://github.com/user-attachments/assets/d4c04486-6d4d-42d8-9274-97752a9e9917)